parser/src/function.rs

```diff
@@ -349,7 +350,25 @@ impl<'input> InlinedFunction<'input> {
         Function::from_offset(hash, self.abstract_origin)
     }
 
-    /// The size of the inlined function.
+    /// The address of the inlined function instance.
+    #[inline]
+    pub fn address(&self) -> Option<u64> {
```

Comment:
I don't think it makes sense to associate a single address with an `InlinedFunction`, since it doesn't have an entry point like a `Function` does.

Reply:
Done (added ranges field instead).